snoop/migrations/0007_auto_20161026_1439.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.9 on 2016-10-26 14:39
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+from django.conf import settings
+
+
+DEFAULT_COLLECTION_DESCRIPTION = "This collection was automatically " \
+                                 "created by the snoop setup."
+
+
+def forward(apps, schema_editor):
+    db_alias = schema_editor.connection.alias
+
+    Document = apps.get_model('snoop', 'Document')
+    document_count = Document.objects.using(db_alias).count()
+    if document_count > 0:
+        Collection = apps.get_model('snoop', 'Collection')
+        try:
+            es_index = settings.SNOOP_ELASTICSEARCH_INDEX
+            path = settings.SNOOP_ROOT
+        except AttributeError:
```

Comment:
You don't need to catch the exception; the error will be pretty self-explanatory.